etl/scripts/faostat/create_new_steps.py

```diff
@@ -795,6 +798,58 @@ def apply_custom_rules_to_list_of_steps_to_create(
     return step_names
 
 
+def create_file_with_latest_versions(dag_steps: Dict[str, Set[str]]) -> None:
+    """Create a csv file in the new version folder, that keeps track of all relevant channel-dataset-versions for the
+    current update.
+
+    This file of versions is necessary because, in the future, when running old steps, they shouldn't simply load the
```

Comment:
This is good solution, but I'm wondering whether managing versions with git somehow would be doable (and simplify this). If we wanted to recreate an old version we'd checkout old commit and run it. There'd be many complications for sure, but perhaps it's something we should consider thinking about?

(Nothing to do in this PR, just dumping my thoughts)